src/Npgsql/TypeHandlers/InternalTypesHandlers/OidVectorHandler.cs

```diff
@@ -46,12 +46,12 @@ internal override NpgsqlTypeHandler Create(PostgresType pgType, NpgsqlConnection
     /// An OIDVector is simply a regular array of uints, with the sole exception that its lower bound must
     /// be 0 (we send 1 for regular arrays).
     /// </summary>
-    class OIDVectorHandler : ArrayHandler<uint>
```

Comment:
`OID` (all-caps) is used in many places around the Npgsql code base (it's even present in Npgsql.sln.DotSettings, which is understood by Jetbrains Resharper and Rider).

If you really want to change this, it needs to happen across the entire codebase, probably in a separate PR.

Reply:
Does it mean that all other renames should be done in an additional PR?

Reply:
Yeah, this PR changes `OID` -> `Oid` but only when it appears in type handler names, which doesn't make much sense to me... You can find `OID` in many other places. Also, this renames the namespace `InternalTypeHandlers` to `InternalTypesHandlers`, which I don't think is good (`InternalTypeHandlers` is better).

So if you really want to do `OID` -> `Oid` in the entire codebase, you can either convert this PR to do that or close this PR and submit another. I'd be OK with this change, although I'd also be OK with leaving things as they are - I don't think it's very important...